src/main/drivers/light_led.c

```diff
@@ -62,35 +56,37 @@ void pgResetFn_statusLedConfig(statusLedConfig_t *statusLedConfig)
 #ifdef LED2_INVERTED
     | BIT(2)
 #endif
-    ;
-}
+    ,
+);
 
 void ledInit(const statusLedConfig_t *statusLedConfig)
 {
     ledInversion = statusLedConfig->inversion;
-    for (int i = 0; i < STATUS_LED_NUMBER; i++) {
-        if (statusLedConfig->ioTags[i]) {
-            leds[i] = IOGetByTag(statusLedConfig->ioTags[i]);
+    for (int i = 0; i < (int)ARRAYLEN(leds); i++) {
+        leds[i] = IOGetByTag(statusLedConfig->ioTags[i]);
+        if (leds[i]) {
             IOInit(leds[i], OWNER_LED, RESOURCE_INDEX(i));
             IOConfigGPIO(leds[i], IOCFG_OUT_PP);
-        } else {
-            leds[i] = IO_NONE;
         }
+        ledSet(i, false);
```

Comment:
Won't we need
```suggestion
        ledSet(i, ledInversion);
```

Reply:
`ledSet` handles inversion internally, it will turn led off